packages/docs/src/components/TableOfContents.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'gatsby';
+import { TableOfContentsItem } from '../helpers/graphQLTypes';
+
+export interface TableOfContentsProps {
+  data: TableOfContentsItem[];
+}
+
+/**
+ * The Desktop version of the table of contents
+ */
+const TableOfContents = ({ data }: TableOfContentsProps) => {
+  return (
+    <div className="c-table-of-contents">
+      <h2 className="c-table-of-contents__heading ds-u-margin-y--0 ds-u-font-size--base">
+        On this page{' '}
+      </h2>
+      <ul className="c-table-of-contents__list ds-u-padding-left--0 ds-u-padding-right--0">
```

Comment:
Let's update `ds-u-padding-left--0` to be `ds-u-padding-left--2` to indent the list a bit under the header